### ISSUE TITLE
URL names in dashboard are not HTML escaped

### ISSUE THREAD

**Issue by kumar303** [locustio/locust#374]
URL names (for stats) are not HTML-escaped in the dashboard. This makes names with angle brackets disappear. For example:

```
self.client.get(url, name='/some-resource/upload/<uuid>')
```

would show up as `/some-resource/upload/` instead of `/some-resource/upload/<uuid>` which is confusing. There could be some XSS security concerns here as well.

**Comment by cyberw** [comment]
Closing due to lack of activity. Let me know if this is still an issue.

**Comment by heyman** [comment]
I just tested this. It's still an issue, and I think we should fix it.

**Comment by peterdemin** [comment]
Can I take it?

**Comment by cyberw** [comment]
@peterdemin By all means!

**Comment by peterdemin** [comment]
Alright, where do I start? I'm experienced with Python, but new to Locust.

**Comment by cyberw** [comment]
Uhm, if you've never used locust then you would need to start using it first, enough that you can at least reproduce the error.

I'm not even sure where the fix should be applied. Maybe in request_stats() in web.py, on line 111.

**Comment by peterdemin** [comment]
I'm using locust here and there for a few years already :-) Thanks for the guidance.

**Comment by cyberw** [comment]
Cool, I was confused and thought maybe the hacktoberfest tag had attracted some kind of "drive by contributors" who didnt know what Locust was even for :P

**Comment by peterdemin** [comment]
No, I just felt the urge to contribute back to the project I enjoy and used the Hacktoberfest label to filter low-hanging fruits.

**Comment by peterdemin** [comment]
Fixed in #1119

**Comment by peterdemin** [comment]
This can be closed, I believe.

**Comment by cyberw** [comment]
Thx!
